qt/lib/include/verovio-qt/qtscenegraphdevicecontext.h

```diff
@@ -0,0 +1,191 @@
+/////////////////////////////////////////////////////////////////////////////
+// Name:        qtscenegraphdevicecontext.h
+// Author:      Jonathan Schluessler
+// Created:     2017
+// Copyright (c) Authors and others. All rights reserved.
+/////////////////////////////////////////////////////////////////////////////
+
+#ifndef __QTSCENEGRAPHDEVICECONTEXT_H__
+#define __QTSCENEGRAPHDEVICECONTEXT_H__
+
+//----------------------------------------------------------------------------
+
+#include <QQuickItem>
+#include <QSGNode>
+#include <QStack>
+#include <QStringList>
+
+//----------------------------------------------------------------------------
+
+#include "devicecontext.h"
+
+namespace vrv_qt {
```

Comment:
change to `vrvQt` and also match the directory to ./include/vrvqt (and not verovio-qt)